core/storage/job/gae_models.py

```diff
@@ -68,7 +68,8 @@ def get_new_id(cls, entity_name):
     # jobs.
     metadata = ndb.JsonProperty(indexed=False)
     # The output of the job. This is only populated if the job has status code
-    # STATUS_CODE_COMPLETED, and is None otherwise.
+    # STATUS_CODE_COMPLETED, and is None otherwise. If populated, this is
+    # expected to be a list of strings.
```

Comment:
(a) What about existing job model instances in production?
(b) Should this be a repeated TextProperty?